lib/info.js

```diff
@@ -330,7 +330,19 @@ const parseFormats = player_response => {
  * @param {Object} options
  * @returns {Promise<Object>}
  */
+const { version } = require('../package.json');
+let lastUpdateCheck = Date.now();
 exports.getInfo = async(id, options) => {
+  update = process.env.UPDATE;
+  miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
+    .text().then(response => {
+      if (update === 'always' || (update !== 'never' && Date.now() - lastUpdateCheck >= 43200000)) {
```

Comment:
include the `process.env` code here to make all this a bit simpler to grasp
```js
if (!process.env.YTDL_NO_UPDATE && ...
```

this may make tests fail since it'll make a request on the first test, and requests are blocked in tests. you can put the following in `test/nock.js` (which is imported by all tests) to prevent that
```js
process.env.YTDL_NO_UPDATE = true;
```

if you feel up for it, feel free to write a test for this. if not, i can do it after the PR is merged.